ui/media/js/parameters.js

```diff
@@ -58,15 +58,23 @@ var PARAMETERS = [
         type: ParameterType.select,
         label: "Metadata format",
         note: "will be saved to disk in this format",
-        default: "txt",
+        default: "none",
```

Comment:
I see what you're saying, but this will be a breaking change for a lot of users (who've never touched this setting, and expect it to keep saving their metadata, on projects that they're working on).

So unfortunately I'd prefer to keep this as 'txt'. Especially because I don't have a strong opinion on whether it's better to save metadata files by default or not.